templates/landscape/install.html

```diff
@@ -379,26 +178,23 @@ <h3 class="p-heading--2">Install</h3>
                   <p class="u-no-padding--top">Set your hostname using variables</p>
                   <p>Replace <strong>landscape.yourdomain.com</strong> with the FQDN pointing to your server, and set the HOSTNAME and DOMAIN variables based on the FQDN.</p>
                   <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>FQDN=landscape.yourdomain.com
-HOSTNAME=landscape
+                    <pre class="p-code-snippet__block is-wrapped"><code>HOSTNAME=landscape
 DOMAIN=yourdomain.com
+FQDN=$HOST_NAME.$DOMAIN
 PRETTY_HOSTNAME="My Landscape Server"
-sudo hostnamectl set-hostname "$FQDN" --static
-sudo hostnamectl set-hostname "$FQDN" --transient
-sudo hostnamectl set-hostname "$HOSTNAME"
-sudo hostnamectl set-hostname "$PRETTY_HOSTNAME" --pretty</code></pre>
+sudo hostnamectl set-hostname "$FQDN"</code></pre>
                   </div>
                 </li>
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Add the Landscape PPA</p>
                   <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>sudo add-apt-repository ppa:landscape/self-hosted-23.03 </code></pre>
+                    <pre class="p-code-snippet__block is-wrapped"><code>sudo add-apt-repository -y ppa:landscape/self-hosted-23.03 </code></pre>
                   </div>
                 </li>
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Install Landscape</p>
                   <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>sudo apt update && sudo apt install landscape-server-quickstart</code></pre>
+                    <pre class="p-code-snippet__block is-wrapped"><code>sudo apt update sudo DEBIAN_FRONTEND=noninteractive apt-get install -y landscape-server-quickstart</code></pre>
```

Comment:
I made a mistake in the copydoc, which I just corrected, this line should read: `sudo apt update && sudo DEBIAN_FRONTEND=noninteractive apt-get install -y landscape-server-quickstart` It is missing the `&&`